src/sonic-host-services/scripts/hostcfgd

```diff
@@ -1245,6 +1245,11 @@ def main():
     signal.signal(signal.SIGTERM, signal_handler)
     signal.signal(signal.SIGINT, signal_handler)
     signal.signal(signal.SIGHUP, signal_handler)
+
+    # Initialize swsscommon signal helper, because python signal handler will be blocked by long running c++ code.
+    # For more information please check: https://docs.python.org/3/library/signal.html
+    SignalHandlerHelper.registerSignalHandler(SIGNAL_INT)
```

Comment:
Also, on reboot, by default systemd sends SIGTERM but not SIGINT. So, this will not handle delay on reboot. In addition to this, we should probably extend `hostcfgd.service` with `KillSignal=SIGINT` or `ExecStop=` that pkill with both `SIGTERM` and then `SIGINT`.

Reply:
either
```
[Service]
Type=simple
ExecStart=/usr/local/bin/hostcfgd
KillSignal=SIGINT
```
or
```
[Service]
Type=simple
ExecStart=/usr/local/bin/hostcfgd
ExecStop=/bin/bash -c '/usr/bin/killall -15 hostcfgd && /usr/bin/killall -2 hostcfgd'
```

Reply:
Thanks, currently swsscommon only handle SIGINT, so will add SIGTERM support with this PR: https://github.com/Azure/sonic-swss-common/pull/626

I will update this PR after swsscommon updated.